freezer.tex

```diff
@@ -151,11 +151,86 @@ \section{The Freezer}
+
+When you do this, the file browser will appear:
+
+\begin{center}
+\includegraphics[trim= 10mm 20mm 10mm 20mm,clip,width=0.7\linewidth]{images/d81-file-browser.png}
+\end{center}
+
+The file browser will list all D81 images on the microSD card. The MEGA65 prioritises the external microSD card, so if you
+prefer to use an image on the internal SD card, the microSD card should be removed (when doing this, the MEGA65 needs
+to be reset as well).
+
+Press the cursor keys to choose a file. If the currently selected file isn't a valid D81, the border will be red,
+otherwise it will be blue, as shown above. Whilst browsing for D81 files, the list of files within the currently
+selected D81 file will be shown on the right-hand side.
+
+When you find the D81 file you wish to mount, press \specialkey{RETURN}.
+This will take you back to the Freezer menu, and the selected D81 file will be shown under the selected drive.
+In the screenshot below, the  \screentext{MEGA65.D81} file has been mounted to drive 0:
+
+\begin{center}
+\includegraphics[trim= 10mm 20mm 10mm 20mm,clip,width=0.7\linewidth]{images/freezer-chosen-d81.png}
+\end{center}
+
+Next, press \megakey{F3} to exit the Freezer menu and return to the currently running program. Alternatively,
+pressing \megakey{F5} will exit the Freezer, and reset the MEGA65 as well.
+
+Now that the file has been mounted, you can use
+the {\bf DIR} command to list the files in the image, or any of the {\bf LOAD}, and {\bf SAVE} commands, like you
+would on a physical disk.
+
+You may have noticed that the file browser has a \screentext{ - NO DISK - } and \screentext{ - INTERNAL 3.5" - }
+entry. These are used for un-mounting a D81 image - but with a difference. \screentext{ - NO DISK - } is used to tell
+the MEGA65 that no disk is inserted, and to not try and find one when performing a disk-based operation. On the other
+hand, \screentext{ - INTERNAL 3.5" - } is used when using a physical 3.5" disk in the internal drive. If no disk is
+inserted and a disk-based operation is performed with the \screentext{ - INTERNAL 3.5" - } option, the MEGA65 will
+try and find a disk, which could take a few seconds.
+
+
+\subsection{Auto booting from a D81 file}
+If you would like the MEGA65 to automatically perform any BASIC commands upon startup, you can create an
+\screentext{AUTOBOOT.C65} PRG file on your disk image. In this file, you can write BASIC programs to do things such
+as changing the {\bf BACKGROUND}, {\bf BORDER}, or {\bf FOREGROUND} colours, or even {\bf LOAD} a program.
+
+For example, to change the background and border colours to black, and load a program called \screentext{MYPROGRAM}
+automatically, create a BASIC listing like the following, then {\bf SAVE} it:
+
+
+\begin{tcolorbox}[colback=black,coltext=white]
+\verbatimfont{\codefont}
+\begin{verbatim}
+10 BACKGROUND 0
+20 BORDER 0
+30 LOAD "MYPROGRAM.PRG"
+SAVE "AUTOBOOT.C65"
+\end{verbatim}
+\end{tcolorbox}
+
+\subsection{Saving D81 file settings}
```

Comment:
I got a little confused as to what was meant by 'D81 file settings'. After reading the paragraph through, it feels more like the intent here was `Selecting the default D81 image to load on startup`.

I also wonder if we should make mention of m65-config setting to permit the user to default to always using the physical drive by default, rather than the virtual drive with 'MEGA65.D81' mounted (but perhaps you might feel it's covered in the reference already).

Reply:
I named it that because the intention was to show to save a default d81 file. The configure screen in question here lets you choose a d81, or just to use the internal drive.